aws/resource_aws_lex_slot_type_test.go

```diff
@@ -0,0 +1,146 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lexmodelbuildingservice"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccLexSlotType(t *testing.T) {
+	resourceName := "aws_lex_slot_type.test"
+	testID := acctest.RandStringFromCharSet(8, acctest.CharSetAlpha)
+	testSlotTypeID := "test_slot_type_" + testID
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsLexSlotTypeDestroy(testSlotTypeID, "$LATEST"),
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccAwsLexSlotTypeConfig, testID),
```

Comment:
I left a suggestion below in using `%q` for the config. If you accept that suggestion then you will need to change `testId` to the expected name `testSlotTypeID`

```suggestion
				Config: fmt.Sprintf(testAccAwsLexSlotTypeConfig, testSlotTypeID),
```